src/Geometry.h

```diff
@@ -135,17 +136,16 @@ class Geometry {
   Point* getFSRPoint(int fsr_id);
   std::string getFSRKey(LocalCoords* coords);
 #ifndef CUDA
-  std::unordered_map<std::size_t, fsr_data> getFSRKeysMap();
+  ParallelHashMap<std::size_t, fsr_data*>* getFSRKeysMap();
 #endif
 
   /* Set parameters */
   void setFSRsToMaterialIDs(std::vector<int> FSRs_to_material_IDs);
   void setFSRsToKeys(std::vector<std::size_t> FSRs_to_keys);
-  void setNumFSRs(int num_fsrs);
   void setCmfd(Cmfd* cmfd);
 
 #ifndef CUDA
-  void setFSRKeysMap(std::unordered_map<std::size_t, fsr_data> FSR_keys_map);
+  void setFSRKeysMap(ParallelHashMap<std::size_t, fsr_data*>* FSR_keys_map);
```

Comment:
Per our convo on the closed source side about exposing this to the user, how about we add it as a SWIG ``%ignore`` to the SWIG interface file? See an example of the ``%ignore`` in use in @samuelshaner's PR #144 [here](https://github.com/mit-crpg/OpenMOC/pull/144/files#diff-d2dacb1ed6c26bddff18021b37dd9c1eR56)